docs/MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+## Maintaining VTEX Toolbelt
+
+This document is for people working on VTEX as Toolbelt maintainers.
+
+### Forcing a new version
+
+Toolbelt has a way of enforcing users to use a minimum version choosed by us. This enforcement is done on the client side (can be disabled by an environment variable, check [EnvVariables](../src/lib/constants/EnvVariables.ts)) and on `vtex.builder-hub` - users using an older toolbelt version will not be allowed to link or publish. This feature is useful for enforcing users to update to the latest version, which can be important when we launch a new version with a major bug fix for example.
+
+Here's how you enforce minumum toolbelt version:
+
+1. You'll have to install the [toolbelt-config-cli](https://github.com/vtex/toolbelt-config-cli):
+
+```
+yarn global add @vtex/toolbelt-config-cli
+```
+
+2. Login to `vtex` account.
+3. Check the current version config:
+
+```
+$ toolbelt-conf config:get
+
+All configs
+{
+  minimumToolbeltVersion: '2.102.1',
+  minimumToolbeltPrereleaseVersion: '2.102.1'
+}
+```
+
+The `minimumToolbeltPrereleaseVersion` is used for enforcing minimum `-beta` versions; `minimumToolbeltVersion` is used for enforcing minimum stable version. You'll probably want to maitain the property `minimumToolbeltPrereleaseVersion <= minimumToolbeltVersion`.
+
+4. Set the version to the one you want to enforce.
+
+If you want to change `minimumToolbeltVersion`:
+
+```
+$ toolbelt-conf config-set -n minimumToolbeltVersion -v 2.92.0
+```
+
+If you want to change `minimumToolbeltPrereleaseVersion`:
+
+```
+$ toolbelt-conf config-set -n minimumToolbeltPrereleaseVersion -v 2.92.0
+```
+
+That's it, now you can check on splunk the latest config changes (all changes are logged):
+
+```
+index=io_vtex_logs app=vtex.toolbelt-config-server@* data.event!=NULL workspace=master
+```
+
+### Deprecating a version
+
+Toolbelt has a way of enforcing users to not use deprecated versions. From time to time toolbelt checks the `npm` registry checking if the current version is deprecated - if it's the user won't be allowed to run commands and will be presented a message like the following:
+
+```
+This version 2.93.0 was deprecated. Please update to the latest version: 'yarn global add vtex'.
+```
+
+This enforcement is on the client side and can be disabled on a execution by setting an environment variable (check check [EnvVariables](../src/lib/constants/EnvVariables.ts)).
+
+There's a problem with this approach - when we deprecate a npm package version, the `latest` tag is not automatically relocated, we have to do it manually, so the full process of deprecating a version is the following (the version chosen is just an example):
+
+1. Deprecate it:
+```
+npm deprecate vtex@2.92.0 -m "Message explaining why this version was deprecated, for example: Bug when sending files to link"
```

Comment:
I tested in a test package and got an error, I thing the correct way is without `-m`
npm deprecate vtex@2.92.0 "message here"